src/ray/common/BUILD

```diff
@@ -0,0 +1,255 @@
+load("//bazel:ray.bzl", "cc_library")
+
+cc_library(
+    name = "constants",
+    hdrs = ["constants.h"],
+)
+
+cc_library(
```

Comment:
you do not have to do it now, but ideally long-term, you would want to have unit tests for most of these individual `cc_library`'s

otherwise, the benefit that the change gives you can be a bit limited. like you do not know if there are `extern` variables being declared somewhere, and you might be missing dependencies for a library.

Reply:
I agree with this. I think the issue here is that we don't have enough unit tests to cover everything :(